backend/controllers/searchWithFilter.test.js

```diff
@@ -38,10 +39,11 @@ describe('search factory', () => {
       getCaseloadsApi.mockReset()
       getSearchableRolesApi.mockResolvedValue([{ roleName: 'Access Role Admin', roleCode: 'ACCESS_ROLE_ADMIN' }])
       getCaseloadsApi.mockResolvedValue([{ text: 'Moorland HMP', value: 'MDI' }])
-      findUsersApi.mockResolvedValue({ searchResults: [], number: 0, page: 0 })
+      findUsersApi.mockResolvedValue({ searchResults: [], totalElements: 20, number: 0, page: 0 })
```

Comment:
Maybe should be 0 so it matches the search results array size?

Reply:
changed to zero